scripts/bulk-password-update-export.js

```diff
@@ -106,30 +127,49 @@ const loadUsers = async () => {
   }
 };
 
-const changeUserPass = async (user, postOptions) => {
-  const newPass = await generatePassword();
-  postOptions.body = {
-    password: newPass
-  };
-  postOptions.uri = `${options.uri}/${user}`;
-  try {
-    await rpn.post(postOptions);
-    console.log('SUCCESS', user, newPass);
-  } catch (e) {
-    console.log('ERROR', user, e.message);
+
+// Thanks https://stackoverflow.com/a/40672956
+String.prototype.splitCSV = function() {
```

Comment:
:warning: Don't update the prototype... ever.

https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#do-not-extend-built-in-prototypes

It's just as easy to call a regular named function.

Reply:
Hah! I see sonar bet me to it!